src/PAModel/Entropy.cs

```diff
@@ -85,6 +85,11 @@ internal class PropertyEntropy
         /// </summary>
         public bool? WasLocalDatabaseReferencesEmpty { get; set; }
 
+        /// <summary>
+        /// Tracks whether TestStepsMetadata is empty or not.
+        /// </summary>
+        public bool DoesTestStepsMetadataExist { get; set; }
```

Comment:
I think this needs to be a nullable `bool`, similar to `WasLocalDatabaseReferencesEmpty` because it is possible that someone would be packing up a Power App which was previously unpacked before this new entropy value was created. You'll also need to make sure you handle that case in your code that reads in the value.